token/class.go

```diff
@@ -41,3 +41,7 @@ func IsOctadecimal(c byte) bool {
 func IsHexadecimal(c byte) bool {
 	return IsDecimal(c) || 'a' <= c && c <= 'f' || 'A' <= c && c <= 'F'
 }
+
+func IsIdent(c byte) bool {
```

Comment:
__[golint]__
* [golint] exported function IsIdent should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->